Assets/Src/AuthController/REST/REST Response DTOs/GoogleRefreshTokenResponse.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Src.AuthController.REST.REST_Response_DTOs
+{
+    [Serializable]
+    public class GoogleRefreshTokenResponse
+    {
+        [JsonProperty("access_token")]
+        public string accessToken;
```

Comment:
As far as I know it is possible to use properties instead of public members if you are using serialization from Newtonsoft.